libs/src/AudiusLibs.ts

```diff
@@ -491,17 +578,17 @@ class AudiusLibs {
         this.ethContracts,
         this.identityService,
         this.solanaWeb3Manager,
+        // @ts-expect-error -- broken for some reason
```

Comment:
hmmmm

Reply:
yeah super weird one, can add a better message for it